x/participation/simulation/simulation.go

```diff
@@ -74,69 +69,6 @@ func SimulateMsgParticipate(
 	}
 }
 
-func SimulateCreateAuction(
```

Comment:
This was just leftover from when `fundraising` didn't support simulation.  I can remove this change from the PR since this removal isn't necessarily relevant for it.  I must have removed when i was making other sim fixes.